fiware_cloto/cloto/wsgi.py

```diff
@@ -65,7 +65,11 @@
 # Creating initial data
 s = ServerInfo(id=1, owner=settings.OWNER, version=settings.VERSION,
                runningfrom=runningfrom, doc=settings.API_INFO_URL)
-s.save()
+try:
+    s.save()
+except Exception as ex:
+    logger.error(ex)
```

Comment:
I suggest to put some details together with ex... due to you know what you are doing here.